fs: Implement fstab devicetree integration for EXT2 (#93767)
zephyrproject-rtos/zephyr: samples/subsys/fs/ext2_fstab/README.rst

.. zephyr:code-sample:: ext2-fstab
   :name: EXT2 filesystem fstab
   :relevant-api: file_system_api

   Define ext2 filesystems in the devicetree.

Overview
***********

> This sample shows how to define a ext2 fstab entry in the devicetree.

Reviewer: ```suggestion
This sample shows how to define an ext2 fstab entry in the devicetree.
```

> This scenario uses a ext2 on a RAM disk. The sample is run on the
> ``native_sim`` board.

Reviewer: ```suggestion
The sample is run on the``native_sim`` board and RAM disk is used for ext2 storage.
```

Author: Updated.
